.github/workflows/ci.yml

```diff
@@ -418,6 +418,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - id: node-cache
```

Comment:
How does job depend on Node.js exactly?

Reply:
We rebuild Keycloak on the EC2 instance so that we can run the Aurora tests, so we have seen the same periodic resolution errors the caching aims to solve on the GH actions. The solution here is to re-use the existing GH runner cache and then transfer a tar of the m2 dependencies to the EC2 runner.